### ISSUE TITLE
CIRAM (nametables) is read-only

### ISSUE THREAD

**Issue by getCursorsExe** [TASEmulators/BizHawk#3676]
Core: QuickNES
Feels inconsistent.

**Comment by nattthebear** [comment]
This is unlikely to be changed.  The way QuickNES implements things, this is a sort of "PPU Bus", (which is why it's 4K and not 2K), and the machinery to make it fully accurate on arbitrary reads as well as arbitrary writes doesn't exist.  The QuickNES strategy with nametables and mirroring was to build something accurate enough to handle many common use cases easily, and then to ignore the rest.

@getCursorsExe What are you trying to accomplish by writing to this domain?  Maybe there's another way to do it.

**Comment by YoshiRulz** [comment]
@SergioMartin86 I see you marked several domains as pokeable, including this one. Did you ensure those pokes are always safe, or did you simply unblock them? "Safety" in this context meaning that the core doesn't segfault and take down EmuHawk, nor does it corrupt itself in such a way that a core reboot won't fix it.

**Comment by SergioMartin86** [comment]
> @SergioMartin86 I see you marked several domains as pokeable, including this one. Did you ensure those pokes are always safe, or did you simply unblock them? "Safety" in this context meaning that the core doesn't segfault and take down EmuHawk, nor does it corrupt itself in such a way that a core reboot won't fix it.

I'm afraid I did not consider any possible side effects before I simply unblocked them. To me addressing seemed safe enough at the moment (see: https://github.com/SergioMartin86/quickerNES/blob/b6e742d5e3fe46e96fce2b198d229201e2967837/source/quickerNES/core/core.hpp#L939), to never access unallocated memory.  I have just did a quick test (poke every single formerly read-only memory byte) and nothing broke. After a reset, all came back to normal. 

Do you have or remember any test case (a game or test rom) we can check this with?

**Comment by YoshiRulz** [comment]
No but poking every address is a good enough test IMO. Calling this done.
